src/shared/Microsoft.Git.CredentialManager/Authentication/MicrosoftAuthentication.cs

```diff
@@ -44,19 +43,65 @@ public class MicrosoftAuthentication : AuthenticationBase, IMicrosoftAuthenticat
             "live", "liveconnect", "liveid",
         };
 
+        #region Broker Initialization
+
+        public static bool IsBrokerInitialized { get; private set; }
+
+        public static void InitializeBroker()
+        {
+            if (IsBrokerInitialized) return;
+
+            IsBrokerInitialized = true;
+
+            // Broker is only supported on Windows 10
+            if (!PlatformUtils.IsWindows10()) return;
+
+            // Nothing to do when not an elevated user
+            if (!PlatformUtils.IsElevatedUser()) return;
```

Comment:
Your time working in the Git codebase is showing.

```suggestion
            if (IsBrokerInitialized)
            {
                        return;
            }

            IsBrokerInitialized = true;

            // Broker is only supported on Windows 10
            if (!PlatformUtils.IsWindows10())
            {
                        return;
            }

            // Nothing to do when not an elevated user
            if (!PlatformUtils.IsElevatedUser())
            {
                        return;
            }
```

Reply:
There are select parts in the GCM codebase that I've used this style of `if` without braces, and it's usually exclusively been for `if (condition) return` early exits.

But you're right. I probably should keep the codebase "clean" :)

Reply:
I guess if it is an existing pattern in this codebase, then that is probably fine. Any C# linter would probably complain, though.

Reply:
`grep -rn "if\s(.*;$" src` returns 37 matches (including the ones added in this PR)